cluster-operator/src/main/java/io/strimzi/operator/cluster/ClusterOperatorConfig.java

```diff
@@ -539,6 +550,11 @@ public String toString() {
                 ",dnsCacheTtlSec=" + dnsCacheTtlSec +
                 ",podSetReconciliationOnly=" + podSetReconciliationOnly +
                 ",podSetControllerWorkQueueSize=" + podSetControllerWorkQueueSize +
+                ",operatorName=" + operatorName +
                 ")";
     }
+
+    public String getOperatorName() {
+        return operatorName;
+    }
```

Comment:
Maybe you can keep the getter together with the other getters instead of having it as the only one after the `toString()` method?

Reply:
:+1: